app/src/main/java/com/bootcamp_android/parking_app/fragments/Main.kt

```diff
@@ -0,0 +1,60 @@
+package com.bootcamp_android.parking_app.fragments
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.bootcamp_android.parking_app.R
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [main.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class main : Fragment() {
```

Comment:
The name of the classes should start with Mayus/Uppercase

Reply:
ok